airflow/dags/sync_kernel_to_website.py

```diff
@@ -115,7 +115,14 @@ def fetch_data(endpoint):
     """
     Obtém o JSON do endpoint do Kernel
     """
-    return kernel_connect(endpoint=endpoint, method="GET").json()
+    kwargs = {
+        "endpoint": endpoint,
+        "method": "GET",
+    }
+    kernel_timeout = Variable.get("KERNEL_FETCH_DATA_TIMEOUT", default_var=None)
```

Comment:
Por padrão não deveria ser 10 ou outro valor no lugar de None?

Reply:
@robertatakenaka Aqui o padrão está como `None` para não passá-lo quando não for configurado. Assim, na prática, o timeout assumido será o definido como padrão em `hook.kernel_connect`.